xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/serverinfo/RippledServerInfo.java

```diff
@@ -0,0 +1,299 @@
+package org.xrpl.xrpl4j.model.client.serverinfo;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.common.collect.Range;
+import com.google.common.primitives.UnsignedInteger;
+import com.google.common.primitives.UnsignedLong;
+import org.immutables.value.Value;
+
+import java.math.BigDecimal;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * An implementation of {@link ServerInfo} that conforms to rippled server payloads.
+ */
+@Value.Immutable
+@JsonSerialize(as = ImmutableRippledServerInfo.class)
+@JsonDeserialize(as = ImmutableRippledServerInfo.class)
+public interface RippledServerInfo extends ServerInfo {
+
+  /**
+   * Construct a builder for this class.
+   *
+   * @return An {@link ImmutableRippledServerInfo.Builder}.
+   */
+  static ImmutableRippledServerInfo.Builder builder() {
+    return ImmutableRippledServerInfo.builder();
+  }
+
+  /**
+   * If {@code true}, this server is amendment blocked.
+   *
+   * @return {@code true} if this server is amendment blocked, otherwise {@code false}.
+   */
+  @JsonProperty("amendment_blocked")
+  @Value.Default
+  default boolean amendmentBlocked() {
+    return false;
+  }
+
+  /**
+   * The version number of the running rippled version.
+   *
+   * @return A {@link String} containing the version number.
+   */
+  @JsonProperty("build_version")
+  String buildVersion();
```

Comment:
Let's discuss which properties we could "push up" to an extension of `ServerInfo`.